mne/preprocessing/tests/test_csd.py

```diff
@@ -183,3 +184,26 @@ def test_csd_fif():
         ch.update(coil_type=FIFF.FIFFV_COIL_EEG, unit=FIFF.FIFF_UNIT_V)
         raw_csd._data[pick] = raw._data[pick]
     assert object_diff(raw.info, raw_csd.info) == ''
+
+
+def test_compute_bridged_electrodes():
+    """Test computing bridged electrodes."""
+    # test I/O
+    raw = read_raw_fif(raw_fname).load_data()
+    raw.pick_types(meg=True)
+    with pytest.raises(RuntimeError, match='No EEG channels found'):
+        bridged_idx, ed_matrix = compute_bridged_electrodes(raw)
+
+    # test output
+    raw = read_raw_fif(raw_fname).load_data()
+    idx0 = raw.ch_names.index('EEG 001')
+    idx1 = raw.ch_names.index('EEG 002')
+    raw._data[idx1] = raw._data[idx0]
+    bridged_idx, ed_matrix = compute_bridged_electrodes(raw)
+    assert bridged_idx == [(idx0, idx1)]
+    picks = pick_types(raw.info, meg=False, eeg=True)
+    assert ed_matrix.shape == (raw.times.size // (2 * raw.info['sfreq']),
```

Comment:
can you make explicit that 2 corresponds to default epoch duration. Btw I would use a different number in the test to make sure it's taken into account